### ISSUE TITLE
Figma: P2P page module

### ISSUE THREAD

**Issue by CloudyRowly** [ANU-Student-Connect/student-connect-2024S2#16]
Design P2P page with Figma

This will be a module that can be access through the menu bar on the left side

### Design:

- Should show top title bar
- Should show left side menu bar

- Reference chat platforms like Facebook Messenger or Discord: Split screen into two vertical section:
  - Left section (~1/5 screen width) is to show existing chats, each existing chats should be shown using a banner, with the user name and last message content
  - Right Section (~4/5 screen width) is to show current selected chat
  - Chat bar should show on bottom

[Figma board](https://www.figma.com/design/7d0tEasmPqA96AMdBZoaIW/Figma-basics?node-id=601-11&t=UUBNiXXbYWGDpgwy-1)

**Comment by CloudyRowly** [comment]
@nicelyhorse is this done? If so, please remember to move it to the suitable status as you work on it next time. I have seen some changes to be made:

- Delete extra functions like call, mic recording, file attachment, emoji, camera (we have no plan on adding those features yet)
- Replace the colours with variable names instead

**Comment by nicelyhorse** [comment]
Fully understand the first modification requirement. Sorry to bother again, but I don't particularly understand the meaning of the second requirement. The Hex values ​​of the colors used in the pages I made are consistent with the Hex values ​​of the colors selected during the discussion.

**Comment by CloudyRowly** [comment]
> Fully understand the first modification requirement. Sorry to bother again, but I don't particularly understand the meaning of the second requirement. The Hex values ​​of the colors used in the pages I made are consistent with the Hex values ​​of the colors selected during the discussion.

Yes, they are the same colours, but Figma provide the function of colour variables, which I have added. If you follow the attached screenshot below, you should be able to find the variables I mentioned. If we use variables, we can easily change the color theme of every page in just 1 click, later on you will not want to go over each element and change the color one by one.

PS: I have updated the issue's status for you, do it yourself next time. Starting by moving it to "In progress" when you start fixing the mentioned requirements.

![Image](https://github.com/user-attachments/assets/d0eeaa51-f65e-455e-8016-1d20f3aa6752)

**Comment by nicelyhorse** [comment]
Thank you very much, team leader. I am not very proficient in using GitHub, but I will master it as soon as possible. I will complete these two changes today and tomorrow.

**Comment by CloudyRowly** [comment]
- Move menu bar to the top
- Remove photos, videos in the extra detail panel, as well as name and avatar in that panel as it is redundant

**Comment by nicelyhorse** [comment]
Got it and finish all the requirement, please check

**Comment by CloudyRowly** [comment]
How can user access the detail pane? Maybe add a button?

![Image](https://github.com/user-attachments/assets/0c536984-f471-41b5-8313-1ac2edfc45c0)
